packages/frontend/src/pages/ErrorPage.tsx

```
@@ -0,0 +1,35 @@
+import { Button } from '@/components/ui/button';
+import { isRouteErrorResponse, Link, useRouteError } from 'react-router-dom';
+
+export function ErrorPage() {
+  const error: any = useRouteError();
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)